mantidproject/mantid
# IN16B QENS: relax too strict error

**Description of work.**

This PR relaxes an error to warning in IN16B QENS reduction, as requested.

**Report to:** M. Gonzalez. 

**To test:**

This should now only log warnings instead of raising an error and stopping.
```
ws = IndirectILLReductionQENS(Run='233277.nxs', BackgroundRun='233002.nxs')
```

Files are below, crop `.zip`.

[233002.nxs.zip](https://github.com/mantidproject/mantid/files/2179524/233002.nxs.zip)
[233277.nxs.zip](https://github.com/mantidproject/mantid/files/2179525/233277.nxs.zip)



No issue, no notes.

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
